### PR TITLE
IS-10017: Rewritten concepts.rst

### PR DESCRIPTION
**Pull request by grove**

### CODE REVIEW

File: sesam-client.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    $ pip install -r requirements.txt
+    $ python sesam.py -version
+    sesam version 1.0.0
+
+Configuration
+=============
+
+::
+
+    •   When running the sesam client for the first time, use this commando:
```
Comment (branislavjenco) [inline sesam-client.rst:73]: commando -> command
also, the `sesam init` command isn't in. It was a part of Ashkan's PR dealing with config groups which wasn't merged

Comment (grove) [reply sesam-client.rst:73]: I'll fix the typo, but this document is not really mine. I've just extracted it out of concepts.rst into its own document.

Comment (branislavjenco) [reply sesam-client.rst]: I fixed the typo, but not sure about the command. We should probably take that change from the PR and merge it to sesam-py

File: concepts.rst [hunk 2]
```diff
@@ -9,23 +9,30 @@ Concepts
...
+
+Why?
+----
 
-Sesam produces and consumes streams of data. Each stream contains a number of data entities. Each entity consists of a
-number of name-value pairs and with some special reserved property names. See the :doc:`entity data model <entitymodel>`
-section for more details. The following is a quick example of the shape of entities that are consumed and exposed by Sesam.
+The data hub is the go-to place for data within the enterprise. Integrations no longer have to be point-to-point. Systems can be loosely coupled instead of being tightly coupled, as is the case for direct integrations. With Sesam, individual systems no longer have to depend on other systems being up. It is also a lot easier to replace systems or to perform migrations. Sesam is the active part and will :ref:`schedule <concepts-scheduling-and-signalling>` how and when pipes are run. If a system is down, the pipe will try getting or sending the data once the system is back up.
```
Comment (BaardBouvet) [inline concepts.rst:28]: Could we somehow highlight that we are focusing our efforts on streaming, merging and enabling higher quality master data (aka globals)?

Comment (grove) [reply concepts.rst:28]: I've added a paragraph about this now. See latest commit.

**Review by BaardBouvet — APPROVED** [review]

**Pull request merged**